src/main/java/dynamok/sink/DynamoDbSinkTask.java

```diff
@@ -70,7 +71,26 @@ String topAttributeName(ConnectorConfig config) {
     @Override
     public void start(Map<String, String> props) {
         config = new ConnectorConfig(props);
-        client = new AmazonDynamoDBClient();
+
+            // TODO : Verify functionality of 
+            //      SystemPropertiesCredentialsProvider
+        if (config.accessKeyId == null  ||  config.secretKeyId == null) {
```

Comment:
The only reason I left it in here was in case we wanted to support the credentials coming from system-level property files ... which I can imagine happening in an IAM environment.   

Removing.